fault/system_verilog_target.py

```diff
@@ -652,6 +669,15 @@ def input_wire(name):
     def make_line(text, tabs=0, tab='    ', nl='\n'):
         return f'{tabs*tab}{text}{nl}'
 
+    def link_kratos_lib(self):
+        from kratos_runtime import get_lib_path
+        lib_path = get_lib_path()
+        dst_path = os.path.join(self.directory, os.path.basename(lib_path))
+        if not os.path.isfile(dst_path):
+            os.symlink(lib_path, dst_path)
+        # also add the directory to the current LD_LIBRARY_PATH
+        self.sim_env["LD_LIBRARY_PATH"] = os.path.abspath(self.directory)
```

Comment:
I think this will need to do `DYLD_LIBRARY_PATH` if the platform is mac, if we haven't tested this yet, perhaps we can at least raise an error if we detect mac platform?